webportal/src/app/cluster-view/services/service-info.js

```diff
@@ -57,3 +57,5 @@ const getNodePods = (kubeURL, namespace, nodeList, callback) => {
         }
     });
```

Comment:
help to refine style in `service-info.js`?

Reply:
I added a line module.exports = { getServiceView } at the end.

Reply:
I mean could you pls help to refine other styles in this file, like indent, semicolon, etc,